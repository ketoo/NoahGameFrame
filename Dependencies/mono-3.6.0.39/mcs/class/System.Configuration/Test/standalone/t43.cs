using System;
using System.Collections.Generic;
using System.Collections.Specialized;
using System.Text;
using System.Configuration;
using System.Web;

class T1
{
	static void Main(string[] args)
	{
		try
		{
			NameValueCollection AppSettings = ConfigurationManager.AppSettings;

			for (int i = 0; i < AppSettings.Count; ++i) {
				Console.WriteLine ("AppSettings[{0}] = {1}", i, AppSettings[i]);
			}
		}
		catch (Exception e)
		{
			// Error.
			Console.WriteLine(e.ToString());
		}
	}
}
