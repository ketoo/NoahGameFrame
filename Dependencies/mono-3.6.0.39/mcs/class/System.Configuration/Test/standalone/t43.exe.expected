AppSettings[0] = Hello World!
